src/background/index.js

```diff
@@ -8,10 +8,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
-
-// For now, we need to load "tldts" UMD module to get it in the global scope,
-// especially for the `serp-report` module.
-import '/vendor/tldts/dist/index.umd.min.js';
+import './tldts';
```

Comment:
why a double import?

Reply:
The serp-report from external code still wants the tldts in global scope, this is artificial file to make it available on global (as we do a the same hack in ghostery-webextension).